content/en/docs/reference/networking/virtual-ips.md

```diff
@@ -266,7 +255,7 @@ the node before starting kube-proxy.
 
 When kube-proxy starts in IPVS proxy mode, it verifies whether IPVS
 kernel modules are available. If the IPVS kernel modules are not detected, then kube-proxy
-falls back to running in iptables proxy mode.
+will exit with an error.
```

Comment:
```suggestion
exits with an error. Kubernetes does offer you some features that you can use to help
with that; for example, you can define an
{{< glossary_tooltip text="init containers" term_id="init-container" >}} that ensures the relevant
module are loaded before starting the container for `kube-proxy`.
```
?

Reply:
Can we document the minimum Linux kernel version for IPVS mode (maybe for different flavors, if that matters)?

I'd assumed it was 2.2.0 or something venerable, that we could assume everyone has.